res/xml/colemak.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<keyboard botton_row="false">
+  <row>
+    <key key0="q" key2="1" key4="esc"/>
+    <key key0="w" key1="~" key2="2" key3="\@" key4="!"/>
+    <key key0="f" key1="£" key2="3" key3="\#" key4="€"/>
+    <key key0="p" key2="4" key3="$" key4="f11_placeholder"/>
+    <key key0="g" key2="5" key3="%" key4="f12_placeholder"/>
+    <key key0="j" key1="accent_caron" key2="6" key3="^" key4="accent_trema"/>
+    <key key0="l" key2="7" key3="&amp;"/>
+    <key key0="u" key2="8" key3="*"/>
+    <key key0="y" key2="9" />
```

Comment:
`-` and `_` are missing ! These are pretty important keys. I suggest to put them at the bottom of the `y` key:
```suggestion
    <key key0="y" key2="9" key3="-" key4="_" />
```